docs/get_start.rst

```diff
@@ -169,3 +169,32 @@ BOX configuration can be refered on `Packer BOX Configuration <https://github.co
+--------------------------------------------------------------------
+
+Here we list a table to reflect the relationship of InfraSIM command and standard server command. Note that the InfraSIM command with (*) here is not the CLI command. Use "infrasim -h" can get the help message.
+
+    +-----------------------------------------------+---------------------------------------------------------+
+    |standard server command                        |InfraSIM command                                         |
+    +===============================================+=========================================================+
+    |AC power on a-node                             |infrasim node start a-node                               |
+    +-----------------------------------------------+---------------------------------------------------------+
+    |AC power off a-node                            |infrasim node stop a-node                                |
+    +-----------------------------------------------+---------------------------------------------------------+
+    |dismiss server node a-node                     |infrasim node destroy a-node                             |
+    +-----------------------------------------------+---------------------------------------------------------+
+    |reset a-node                                   |infrasim node restart a-node                             |
+    +-----------------------------------------------+---------------------------------------------------------+
+    |KVM - virtual keyboard, visual monitor         |Connecting to InfraSIM with VNC client(*)                |
+    +-----------------------------------------------+---------------------------------------------------------+
+    |configuration update for a-node                |1. update a-node yaml file(*)                            |
+    |(node type, nic, processor, drive, memory)     |2. infrasim config update a-node [a-node yaml file path] |
+    |                                               |3. infrasim node stop a-node                             |
+    |                                               |4. infrasim node destroy a-node                          |
+    |                                               |5. infrasim node start a-node                            |
+    +-----------------------------------------------+---------------------------------------------------------+
+    |add new server node b-node                     |1. compose b-node yaml file(*)                           |
+    |                                               |2. infrasim config add b-node [b-node yaml file path]    |
+    |                                               |3. infrasim node start b-node                            |
+    |                                               |4. infrasim config list                                  |
+    +-----------------------------------------------+---------------------------------------------------------+
```

Comment:
More rows on:

Check server a-node specification                <--->       infrasim node info a-node
Check server a-node running status                          <--->       infrasim node status a-node
                                                                                                  "a-node-bmc is running" AC is on, bmc's alive
                                                                                                   "a-node-node is running" - it indicates the compute                             node is powered on